src/olympia/migrations/952-correct-index-theme-user-counts.sql

```diff
@@ -0,0 +1,21 @@
+-- Drop old index first, if it exists.
+
+-- Set the index_name or NULL if it doesn't exist
+SET @KEY_ADDON_DATE_IDX := (
+    SELECT INDEX_NAME FROM information_schema.STATISTICS WHERE
+        TABLE_SCHEMA = (SELECT DATABASE()) AND
+        TABLE_NAME = 'theme_user_counts' AND
+        INDEX_NAME = 'addon_date_idx'
+    LIMIT 1);
```

Comment:
the index is a composite key on (addon_id, date) - so it matches two columns in that query. That's why we need the limit 1.